content/php/concepts/math-functions/terms/log10/log10.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: 'log10()'
+Description: 'Returns the base-10 logarithm of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`log10()`** function returns the base-10 logarithm of a number.
+
+## Syntax
+
+```pseudo
+log10($arg);
+```
+
+## Parameters
```

Comment:
I wanted you to remove this Parameters header so the following sentence is part of the Syntax section.